jenkins/jobs/integration_tests.pipeline

```diff
@@ -121,7 +121,13 @@ pipeline {
     success {
       withCredentials([usernamePassword(credentialsId: 'airshipci_city_cloud_openstack_credentials', usernameVariable: 'OS_USERNAME', passwordVariable: 'OS_PASSWORD')]) {
         withCredentials([sshUserPrivateKey(credentialsId: 'airshipci_city_cloud_ssh_keypair', keyFileVariable: 'AIRSHIP_CI_USER_KEY')]){
-          sh "./jenkins/scripts/integration_test_clean.sh"
+          script {
+            if ("{TTL}" != "") {
+              echo "Going to sleep for "{TTL}" hours"
+              sleep(time:"{TTL}",unit:"HOURS")
```

Comment:
Interesting. I'm not sure I like the idea of very long running Jenkins builds to keep resources around. A long running Jenkins job will continue to consume an executor on the agent, even if it is sleeping.

What if instead, we have an option to skip the cleanup? The VM will eventually be cleaned up by the integration tests cleanup job:
https://jenkins.nordix.org/view/Airship/job/airship_master_integration_tests_cleanup

Currently looks like the machines are cleaned up after 6 hours: 
https://github.com/metal3-io/project-infra/blob/cc846fe9835d1373a7eb0954c17b24eaf0d4ec69/jenkins/scripts/integration_clean.sh#L23

Potentially we could be less aggressive here. Or alternatively, you could tag the VM and use `--not-tags` to skip deletion. (We'd also have to do the same for the ports.)

Reply:
I would support the tagging of VMs that were started with the "keep 24h" jobs and the modification of the cleanup script.
We could also limit the number of VMs kept alive with the "keep" commands if we limit the number of active "keep" builds to 1 per job.

Reply:
This sounds a better approach but a bit confused. If we are skipping the clean up and deletion, who will do the cleaning later on (run make clean)? For the VM deletion we have a script that runs periodically and we can make it so that after 24 hours it catches the "keep 24h" VM and deletes it. However, before that we need to run make clean as well, which is part of https://github.com/metal3-io/project-infra/blob/cc846fe9835d1373a7eb0954c17b24eaf0d4ec69/jenkins/jobs/integration_tests.pipeline#L124